src/ECER.Clients.RegistryPortal/ecer.clients.registryportal.client/src/components/pages/Application.vue

```diff
@@ -44,14 +44,28 @@
       </v-btn>
     </template>
     <template #actions>
-      <v-container>
-        <v-btn v-if="showSaveButtons" :loading="loadingStore.isLoading('application_get')" rounded="lg" color="primary" @click="handleSaveAndContinue">
-          Save and continue
-        </v-btn>
-        <v-btn v-if="showSubmitApplication" rounded="lg" color="primary" :loading="loadingStore.isLoading('application_post')" @click="handleSubmit">
-          Submit Application
-        </v-btn>
-      </v-container>
+      <v-window class="my-n10">
+        <v-stepper-window>
+          <v-container>
+            <v-row>
+              <v-col>
+                <v-btn
+                  v-if="showSaveButtons"
+                  :loading="loadingStore.isLoading('draftapplication_put')"
+                  rounded="lg"
+                  color="primary"
+                  @click="handleSaveAndContinue"
+                >
+                  Save and continue
+                </v-btn>
+                <v-btn v-if="showSubmitApplication" rounded="lg" color="primary" :loading="loadingStore.isLoading('application_post')" @click="handleSubmit">
+                  Submit Application
+                </v-btn>
+              </v-col>
+            </v-row>
+          </v-container>
+        </v-stepper-window>
+      </v-window>
```

Comment:
Just curious. This was done so the buttons would line up? Brutal css.

Reply:
Haha yes, it's ugly but I figured the best way to ensure they are aligned is use the same components as the step window above... I could just try and work backwards and determine a x margin but could be hard to make it always align...